ds_judgements_public_ui/templates/includes/results_filters_inputs.html

```diff
@@ -10,104 +10,112 @@
 </div>
 <div class="structured-search__multi-fields-panel">
   <div class="structured-search__limit-to-container">
-    <fieldset>
+    {% if context.errors|has_errors_for_field:"from_date" %}
+      <fieldset class="set-1">
```

Comment:
We verified that this worked locally, but when this is false the html will have a closing fieldset tag without an opening one which of course isn't best practice.

Like I said we verified that it generally looked fine in the UI with the code like this, but we have in fact removed fieldset from the non error path, which means that either fieldset was redundant before or that it had some other importance that wasn't necessarily visual.




We thought it might be possible to have the conditional statement inside the `<fieldset>` tag, but that broke the html /not rendering anything after this line when the condition was true. Maybe we did it wrong, or is that just not allowed in django templates?

An alternative to this, that definitely would work, would  be duplicating the whole fieldset here and wrapping the if block around that and then adjusting the else blocks fieldset to have no `class=set-1` (we did try this initially but got messy due to the fieldset being large and we got a bit tangled, but should be fine if we do it carefully)

Reply:
just noticed that @timcowlishaw has given the correct form of the conditional within the tag itself in the above comment!